src/config/defaultConfig.hpp

```diff
@@ -3,20 +3,13 @@
 #include <string>
 
 inline const std::string AUTOCONFIG = R"#(
-# ██╗  ██╗██╗   ██╗██████╗ ██████╗ ██╗      █████╗ ███╗   ██╗██████╗     
-# ██║  ██║╚██╗ ██╔╝██╔══██╗██╔══██╗██║     ██╔══██╗████╗  ██║██╔══██╗    
-# ███████║ ╚████╔╝ ██████╔╝██████╔╝██║     ███████║██╔██╗ ██║██║  ██║    
-# ██╔══██║  ╚██╔╝  ██╔═══╝ ██╔══██╗██║     ██╔══██║██║╚██╗██║██║  ██║    
-# ██║  ██║   ██║   ██║     ██║  ██║███████╗██║  ██║██║ ╚████║██████╔╝    
-# ╚═╝  ╚═╝   ╚═╝   ╚═╝     ╚═╝  ╚═╝╚══════╝╚═╝  ╚═╝╚═╝  ╚═══╝╚═════╝     
-                                                                       
-# This is an autogenerated Hyprland config file.
+# Autogenerated by Hyprland.                                                                  # This is an autogenerated Hyprland config file.
 # Refer to the wiki (https://wiki.hyprland.org/) for more information.
 
 #! Please note, not all available settings / options are set here.
 #! For a full list, see the wiki (https://wiki.hyprland.org/)
 
-###! REMOVE THIS LINE TO DISABLE THAT BOTHERSOME YELLOW WARNING!
+###! REMOVE THIS LINE TO REMOVE THAT YELLOW WARNING!
```

Comment:
"That yellow warning"? What yellow warning?
Make sure to clearly specify what marking we're talking about. For example
> "Remove this line to disable the yellow warning on top"

Reply:
Noticed this is at the wrong place, but you get the point.